index.html

```diff
@@ -75,6 +75,7 @@
                 Other Pages
               </a>
               <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownBlog">
+                <a class="dropdown-item" href="profile-form.html">Create an account</a>
```

Comment:
This will need to be added to all pages and templates.

Reply:
Good call @profounddark. I coulnd't see the profile form page.